inst/pages/subsetting.qmd

```diff
@@ -180,13 +179,27 @@ as well as Chlamydiae in different sites of the human body,
 
 ## Filtering based on library size {#sec-subset-library-size}
 
-As a preprocessing step, one might want to remove samples that do not
-exceed certain library size, i.e., total number of counts. Additionally,
-sometimes data might contain, samples that do not contain any of the
-features present in the dataset. This can occur, for example, after data
-subsetting. To focus only samples containing sufficient information, we
-might want to remove those instances. In this example, we are
-interested only those features that belong to Species _Achromatiumoxaliferum_.
+As a preprocessing step, one might want to remove samples below
+certain library size, i.e., total number of counts.
+
+```{r}
+#| label: subset_libsize
+
+#Inspect library sizes among samples:
+summary(colSums(assay(tse, "counts")))
```

Comment:
The library size is already calculated, so one can just fetch it from `colData()`. This might be more clear as the calculated library sizes are used in subsetting in line 192

`summary( tse$total )`